### ISSUE TITLE
channel_wise group or filter_wise group

### ISSUE THREAD

**Issue by lippman1125** [astorfi/attention-guided-sparsity#3]
In file losses.py
`grouped_sum = tf.sqrt(tf.reduce_sum(tf.pow(W,2),axis=[0,1,2]))`
I think it's filter wise group, not input channel wise group, but the comment is channel wise group
